lib/graphql/anycable/stats.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module AnyCable
+    # Calculates  amount of Graphql Redis keys
+    # (graphql-subscription, graphql-fingerprints, graphql-subscriptions, graphql-channel)
+    # Also, calculate the number of subscribers grouped by subscriptions
+    class Stats
+      SCAN_COUNT_RECORDS_AMOUNT = 1_000
+
+      attr_reader :redis, :config, :list_prefixes_keys, :include_subscriptions
+
+      def initialize(redis:, config:, include_subscriptions: false)
+        @redis = redis
+        @config = config
+        @include_subscriptions = include_subscriptions
+        @list_prefix_keys = list_prefixes_keys
```

Comment:
It doesn't seem we use this ivar.

Reply:
oh, yea, It was a rudiment.
I didn`t see :(

fixed